### PR TITLE
Fix #25 add support for user-defined VM args

### LINKED ISSUE #25: Expose server JVM launch parameters
We should make it possible for users to override the default JVM launch parameters for the server.
Typically, users should be able to increase the amount of memory  of the JVM or launch the server with some java agents (eg. lombok, as requested on https://marketplace.visualstudio.com/items?itemName=redhat.java)

### PR DESCRIPTION
**Pull request by fbricon**
Signed-off-by: Fred Bricon <fbricon@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,11 @@
 					"type": ["string", "null"],
 					"default": null,
 					"description": "Specifies the folder path to the JDK 8 used to launch the Java Language Server."
+				},
+				"java.jls.vmargs": {
```
Comment (gorkem) [inline package.json]: Should we use java.jdt.ls.vmargs since the upstream project is now renamed to jdt.ls

Comment (fbricon) [inline package.json]: sure we can do that.

I've also been pondering whether we should also rename java.home too. But I think it's not necessary as it's more _universal_

Comment (gorkem) [inline package.json]: agree on java.home, it is more universal and basically overwrites the JAVA_HOME value for the server.

Comment (fbricon) [inline package.json]: Changed to java.jdt.ls.vmargs. Also updated readme and changelog.

**Pull request merged**